docs/examples/identity/readme.md

```diff
@@ -0,0 +1,15 @@
+    #     ___  ____     _    ____ _     _____
+    #    / _ \|  _ \   / \  / ___| |   | ____|
+    #   | | | | |_) | / _ \| |   | |   |  _|
+    #   | |_| |  _ < / ___ | |___| |___| |___
+    #    \___/|_| \_/_/   \_\____|_____|_____|
+***
+## Identity Resource and Datasource Examples
+This example demonstrates the following Identity concepts:
+* Define multiple OCI providers targeting different regions
```

Comment:
I think that we should have a separate place for examples of general concepts like this. Mixing them into the service/resource examples will make them less discoverable. I would image something like:
- docs/examples
  - identity
  - compute
  - other services
  - concepts
    - multi_region
    - filtering

Reply:
I think those concept examples should certainly be represented fully and distinctly, I don't have an issue including something like filtering in these examples though if it makes the files a little more meaningful without adding unnecessary complexity. So multi-region is probably a good removal, though it results in a very basic AD file.

Reply:
agree (a basic AD file seems fine)